### PR TITLE
1074-save-broken-post-refactor

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
Caused by a capitalization mismatch, so convert all `OPFS` to `Opfs`, except for dir-name (to reduce merge issues for now).

Tested preview, and it does do the commit, but reports an error in UI on the load?

image

image

### CODE REVIEW

**Review by nickcastel50 — APPROVED** [review]
LGTM

File: src/Components/Open/SaveModelControl.jsx [hunk 1]
```diff
@@ -191,19 +183,16 @@ function SaveModelDialog({isDialogDisplayed, setIsDialogDisplayed, navigate, org
       setRequestCreateFolder(false)
     }
 
-    setSelectedFolderName('none')
+    setSelectedFolderName('')
```
Comment (pablo-mayrgundter) [inline src/Components/Open/SaveModelControl.jsx:186]: This was getting a react error from selecting an option not in the list

File: src/Components/Open/SaveModelControl.jsx [hunk 2]
```diff
@@ -191,19 +183,16 @@ function SaveModelDialog({isDialogDisplayed, setIsDialogDisplayed, navigate, org
...
 
     setCurrentPath(newPath)
 
-    const {files, directories} = await getFilesAndFolders(repoName, owner, newPath, accessToken)
-    // eslint-disable-next-line no-shadow
-    const fileNames = files.map((file) => file.name)
```
Comment (pablo-mayrgundter) [inline src/Components/Open/SaveModelControl.jsx:200]: and there were some vars showing up in lint for me as unused, and these came out transitively as well

**Review by pablo-mayrgundter — COMMENTED** [review]
Left a couple comments for some fixes for warnings i was getting

**Pull request merged**